### ISSUE TITLE
Permission Issue

### ISSUE THREAD

**Issue by jx99** [Thue/rfwadmin#39]
I am running Redhat on AWS.
Even after changing server.properties permission to 777, i am getting this error:

<br />
<b>Warning</b>:  file_put_contents(/var/lib/minecraft/servers/default/server/server.properties): failed to open stream: Permission denied in <b>/var/lib/minecraft/web/include/libs/properties.php</b> on line <b>157</b><br />
Failed to save server.properties. The web server user probably doesn't have permission to write that file.

**Comment by Thue** [comment]
Did you follow this instruction from the readme?

> You also need to set "SELINUX=permissive" in /etc/sysconfig/selinux

**Comment by jx99** [comment]
Yes, I found it the issue. It was SELINUX. When I disable it, it works fine now.
